java/google/registry/rde/imports/RdeHostImportAction.java

```diff
@@ -77,9 +85,10 @@ public void run() {
   }
 
   /** Mapper to import hosts from an escrow file. */
-  public static class RdeHostImportMapper extends Mapper<HostResource, Void, Void> {
+  public static class RdeHostImportMapper extends Mapper<JaxbFragment<XjcRdeHostElement>, Void, Void> {
```

Comment:
There's a bunch of lines I'm having to wrap to the 100 character limit (this being on).